test/squish/suite_VerifySettingsDialog/tst_VerifySettingsDialog/test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# mypy: disable-error-code="attr-defined"
+
+import gui_helpers
+import test
+
+profileName = "deadlinecloud_squish"
+jobHistDir = "~/.deadline/job_history/deadlinecloud_squish"
+farmName = "Deadline Cloud Squish Farm"
+queueName = "Squish Automation Queue"
+jobAttachments = "COPIED"
+storageProfile = "Squish Storage Profile"
+conflictResOption = "NOT\\_SELECTED"
+loggingLevel = "WARNING"
+
+
+def main():
+    gui_helpers.launchDeadlineConfigGUI()
```

Comment:
Is there a way to structure these in their own individual tests? What does the log output look like when a test run is done? Is it easy to see which tests have failed at a glance?

Reply:
Each line in def main() function is it's own individual test (or what do you mean by this)? 

Squish will print out what test failed (or passed) as well-- attached screenshot when running the Squish test
![Screenshot 2024-07-15 at 3 29 18 PM](https://github.com/user-attachments/assets/1cc79a1e-8790-4b7f-a1bf-9602a93e376d)
 from the Command Line. I've also written these tests so that it should be easy to deduce what specific test failed when the test run is over. Also open to any specific suggestions, etc.